web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -657,7 +657,8 @@ def test_remote_to_local_with_baseline_file(self):
 
         resolved_results, _, returncode = get_diff_results(
             [self._run_names[0]], [baseline_file_path], '--resolved', 'json',
-            ["--url", self._url])
+            ["--url", self._url,
+             "--review-status", "unreviewed", "confirmed", "false_positive"])
```

Comment:
I don't see any test cases which would test that fix dates are set properly on review status changes / storage events. For this reason please create some more test cases and also check my scenarios above.